azurelinuxagent/common/protocol/wire.py

```diff
@@ -867,6 +884,33 @@ def get_header_for_cert(self):
             "x-ms-guest-agent-public-x509-cert": cert
         }
 
+    def write_ip_on_rdma(self):
```

Comment:
Given that this code does not actually write any data to the device, I think the methos is misnamed. setup_rdma_dev() may be more appropriate